### ISSUE TITLE
How to use models in Hugging Face?

### ISSUE THREAD

**Issue by code4luck** [alchemab/antiberta#8]
Hello, I have used the model weights and tokenizer that you uploaded on Hugging Face, but I encountered some diffcults when actually using it. I find the output of tokenizer is weird.Could you tell me Is it necessary to insert a space between every character in the input sequence? and is the use of H and L (on the start of sequence) necessary? thanks!
`    tokenizer = RoFormerTokenizer.from_pretrained("trans_model/antiberta2")
    model = RoFormerForMaskedLM.from_pretrained("trans_model/antiberta2")
    seqs = ["ḢACCKLMNDDDKKLL", "ḶALYYNNMACD"]
    outs = tokenizer(
        seqs,
        return_tensors="pt",
        padding=True,
        truncation=True,
        max_length=1024,
    )`
`
>>> outs
{'input_ids': tensor([[ 1, 26,  4,  2],
        [ 1, 27,  4,  2]]), 'token_type_ids': tensor([[0, 0, 0, 0],
        [0, 0, 0, 0]]), 'attention_mask': tensor([[1, 1, 1, 1],
        [1, 1, 1, 1]])}
`
`    seqs =  ["Ḣ A C C K L M N D D D K K L L", "Ḷ A L Y Y N N M A C D"]`
`
>>> out
{'input_ids': tensor([[ 1, 26,  5,  6,  6, 13, 14, 15, 16,  7,  7,  7, 13, 13, 14, 14,  2],
        [ 1, 27,  5, 14, 24, 24, 16, 16, 15,  5,  6,  7,  2,  0,  0,  0,  0]]), 'token_type_ids': tensor([[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
        [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]]), 'attention_mask': tensor([[1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
        [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0]])}
`
